docs/index.md

```diff
@@ -0,0 +1,41 @@
+# Kili's Python SDK
+
+## What is Kili?
+
+Kili is a platform that empowers a data-centric approach to Machine Learning through quality training data creation. It provides collaborative data **annotation tools** and APIs that enable quick iterations between reliable dataset building and model training. More info [here](https://kili-technology.com/product/label-annotate).
+
+## Requirements
+
+- Python >= 3.7
+- Create and copy a Kili API key
+- Add the `KILI_API_KEY` variable in your bash environment (or in the settings of your favorite IDE) by pasting the API key value you copied above:
+
+```bash
+export KILI_API_KEY='<you api key value here>'
+```
+
+## Installation
+
+Install the Kili client with pip:
+
+```bash
+pip install kili
+```
+
+## Usage
+
+Instantiate the Kili client:
+
+```python
+from kili.client import Kili
+kili = Kili()
+# You can now use the Kili client!
+```
+
+Note that you can also pass the API key as an argument of the `Kili` initialization:
+
+```python
+kili = Kili(api_key='MY API KEY')
+```
+
+Here is a sample of the operations you can do with the Kili client:
```

Comment:
missing end of the document?

Reply:
After the discussion that we had, I wanted to put less information on the home page, just to have a website for the API reference. You can maybe think about what should be on the home page

Reply:
in this case remove the last sentence and we will be good.

Reply:
ah yes, I didn't see that passing!